src/lsst/cm/prod/configs/HSC/test/hsc_weekly.yaml

```diff
@@ -40,7 +40,7 @@ step1:
     prerequisites: []
     includes: ["step"]
     pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/HSC/DRP-RC2.yaml#step1"
-    data_query_base: "instrument = 'HSC' and skymap = 'hsc_rings_v1'"
+    data_query_base: "instrument = 'HSC' and skymap = 'hsc_rings_v1' AND band IN ('g','r','i','z','y')"
```

Comment:
Minor point, but you have "and" in one case but "AND" in the next one; probably want to standardize.